src/AppBundle/Entity/Report/Traits/ReportProfDeputyCostsEstimateTrait.php

```diff
@@ -63,8 +63,7 @@ trait ReportProfDeputyCostsEstimateTrait
         ['typeId' => 'contact-client', 'hasMoreDetails' => false],
         ['typeId' => 'contact-case-manager-carers', 'hasMoreDetails' => false],
         ['typeId' => 'contact-others', 'hasMoreDetails' => false],
-        ['typeId' => 'forms-documents', 'hasMoreDetails' => false],
-        ['typeId' => 'other', 'hasMoreDetails' => true],
```

Comment:
Do we need any migrations to remove these costs from the database?

Reply:
Yes, good spot. We shall need to remove from db on release.